config/plugins.yml

```diff
@@ -3,6 +3,7 @@ loomio_webhooks:
 
 loomio_org_plugin:
   repo:         loomio/loomio_org_plugin
+  branch:       tests-for-sidebar-implementation
```

Comment:
I noticed that you merged my changes at one point, but then it looked like you might have reverted them, so I reimplemented them in a new branch.